templates/project/cargo-generate.toml

```diff
@@ -9,17 +9,7 @@ prompt = "MCU"
 choices = ["esp32"]
 default = "esp32" 
 
-[placeholders.viam]
+[placeholders.fragid]
 type = "string"
-prompt = "Viam's robot config (leave blank if you don't have a robot configured yet)"
-default = ""
-
-[placeholders.ssid]
-type = "string"
-prompt = "Enter wifi name (only 2.4 gHz network are supported)"
-default = "Viam-2G"
-
-[placeholders.pwd]
-type = "string"
-prompt = "Enter the wifi password (leave blank if you don't want to store it)"
-default = ""
+prompt = "Enter fragment ID"
+default = "d385b480-3d19-4fad-a928-b5c18a58d0ed"
```

Comment:
Could we add a comment linking to `https://app.viam.com/fragment?id=d385b480-3d19-4fad-a928-b5c18a58d0ed` so it is easy to jump to the actual fragment definition from the source? Should the fragment actually represented by the URL?

Reply:
We can maybe give more context? I added it here so provisioning has a default config to be set. But in reality we need more discussion about what a default fragment should look like if we want any.
For now Viam app doesn't seem to honor fragment id